### ISSUE TITLE
Disable i18n cache

### ISSUE THREAD

**Issue by mdarveau** [balderdashy/sails#2269]
Hi,

I make sure my application is i18n ready while I develop it. However, the locale file (ie: `en.json`) seems to get cached after a template rendering. This is really annoying when testing that a specific locale fits nicely in the UI.

I am not sure which layer caches this but is there a way to disable this? I am using Jade template.

Thanks!

**Comment by sailsbot** [comment]
Thanks for posting, @mdarveau.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
